packages/frontend/model/extract-config.ts

```diff
@@ -4,6 +4,11 @@ export const extract = (data: {}): ConfigType => ({
     ajaxUrl: getNonEmptyString(data, 'config.page.ajaxUrl'),
     sentryPublicApiKey: getString(data, 'config.page.sentryPublicApiKey', ''),
     sentryHost: getString(data, 'config.page.sentryHost', ''),
+    subscribeWithGoogleApiUrl: getString(
+        data,
+        'config.page.subscribeWithGoogleApiUrl',
```

Comment:
Nothing unique to this PR, but just a note for the team really about whether we should pass this kind of config on every request or even at all? It's not really part of the data model and arguably should be the responsibility of dotcom-rendering to know, not the client.

(But definitely not a blocker to this PR.)

Reply:
I did wonder where this config shoud live. Do you mean for `dotcom-rendering` to own the config `subscribeWithGoogleApiUrl = https://swg.theguardian.com` instead?